include/ppx/application.h

```diff
@@ -297,7 +297,13 @@ struct ApplicationSettings
     struct
     {
         grfx::Api api               = grfx::API_UNDEFINED;
-        bool      enableDebug       = false;
+
+#if !defined(NDEBUG) and !defined(PPX_ANDROID)
```

Comment:
nit: maybe add the comment explaining debug is not enabled by default on Android because validation layers on old version can be a mess to setup.

Reply:
done.